packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/class_filter_dialog.dart

```diff
@@ -0,0 +1,159 @@
+                    tooltip: _showHelp ? 'Hide help' : 'Show help',
+                    icon: _showHelp
+                        ? Icons.keyboard_arrow_up
+                        : Icons.keyboard_arrow_down,
+                    label: 'Help',
+                    onPressed: () => setState(() => _showHelp = !_showHelp),
+                  ),
+                  if (_showHelp) ...[
+                    const SizedBox(height: denseSpacing),
+                    const Text('Choose and customize the filter.\n'
+                        'List full or partial class names separated by new lines. For example:\n\n'
+                        '  package:myPackage/src/myFolder/myLibrary.dart/MyClass\n'
+                        '  MyClass\n'
+                        '  package:myPackage/src/\n\n'
+                        'Specify:\n'
+                        '  - ${ClassFilter.coreLibrariesAlias} for core classes without package prefix\n'
+                        '  - ${ClassFilter.dartAndFlutterLibrariesAlias} for "dart." and "package:" libraries published by Dart and Flutter orgs.'),
+                  ],
+                  SizedBox(height: itemSpacing),
+                  RadioButton<ClassFilterType>(
+                    label: 'Show all classes',
+                    itemValue: ClassFilterType.all,
+                    groupValue: _type,
+                    onChanged: onTypeChanged,
+                  ),
+                  SizedBox(height: itemSpacing),
+                  RadioButton<ClassFilterType>(
+                    label: 'Show all classes except:',
+                    itemValue: ClassFilterType.except,
+                    groupValue: _type,
+                    onChanged: onTypeChanged,
+                  ),
+                  Padding(
+                    padding: EdgeInsets.only(left: textFieldLeftPadding),
+                    child: TextField(
+                      keyboardType: TextInputType.multiline,
+                      maxLines: null,
+                      controller: _except,
+                    ),
+                  ),
+                  SizedBox(height: itemSpacing),
+                  RadioButton<ClassFilterType>(
+                    label: 'Show only:',
+                    itemValue: ClassFilterType.only,
+                    groupValue: _type,
+                    onChanged: onTypeChanged,
+                  ),
+                  Padding(
+                    padding: EdgeInsets.only(left: textFieldLeftPadding),
+                    child: TextField(
+                      keyboardType: TextInputType.multiline,
+                      maxLines: null,
+                      controller: _only,
+                    ),
+                  ),
+                ],
+              ),
+            ),
+      actions: [
+        DialogTextButton(
+          onPressed: () =>
+              setState(() => _loadStateFromFilter(ClassFilter.empty())),
+          child: const Text('Reset Defaults'),
+        ),
```

Comment:
remove this once you use the new common widget I suggested `FilterDialogTitle` (see previous comment)